ballerina/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "1.2.0"
 authors = ["Ballerina"]
 keywords = ["io", "json", "xml", "csv", "file"]
 repository = "https://github.com/ballerina-platform/module-ballerina-io"
+icon = "resources/icon.png"
```

Comment:
Let's move the icon to ballerina root directory

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/444/commits/f1f24833f2115ee2e90d51f7598e8ebab6bd65cd